tests/tests.rs

```diff
@@ -439,3 +469,14 @@ fn test_sse42_chunked_panic() {
         simdutf8::basic::imp::aarch64::neon::ChunkedUtf8ValidatorImp,
     >(b"abcd", 1, true);
 }
+
+// the test runner will ignore this test probably due to limitations of panic handling/threading
+// of that target--keeping this here so that when it can be tested properly, it will
```

Comment:
```suggestion
// the test runner will ignore this test probably due to limitations of panic handling/threading
// of that target--keeping this here so that when it can be tested properly, it will
// FIXME: remove this comment once this works properly.
```